web/client/utils/PrintUtils.js

```diff
@@ -496,7 +497,33 @@ const PrintUtils = {
                         "opacity": getOpacity(layer),
                         "tileSize": [256, 256],
                         "maxExtent": [-20037508.3392, -20037508.3392, 20037508.3392, 20037508.3392],
-                        "resolutions": MapUtils.getResolutions()
+                        "resolutions": [
```

Comment:
What is the difference between this hardcoded list and the list previously returned by MapUtils.getResolutions ?

Reply:
Isn't getGoogleMercatorResolutions in MapUtils returing the same?

Reply:
For getResolutions it is not the same, and this is the cause of the issue. If the map has different resolutions from the server, we have the bug.

Yes I think it is the same for `getGoogleMercatorResolutions`, I see it now that it should.
I've seen the same hardcoded resolutions in osm and other entries of print utils (where I copied it), so I supposed we needed to use that and didn't noticed if we could use it. 
Maybe an improvement is to get them with max as `maxNativeZoom`, but I'm not sure this works in every tile provider in the list.

Reply:
I noticed that certain layers have problem with their maxNative in tests, 
So I should keep this explicitly hardcoded, as osm or mapquest, until we find out a better way to handle it.